postgres-appliance/scripts/configure_spilo.py

```diff
@@ -782,6 +782,11 @@ def write_crontab(placeholders, overwrite):
 
     lines = ['PATH={PATH}'.format(**placeholders)]
 
+    # if SSL_CERTIFICATE_FILE is set then the ceritificate is provided by the
+    # user and might be rotated.
+    if os.environ.get('SSL_CERTIFICATE_FILE'):
+        lines += ['*/5 * * * * PGDATA={PGDATA} /scripts/test_reload_ssl.sh 5'.format(**placeholders)]
```

Comment:
`PGDATA={PGDATA}` -- yeah, cron cleans the environment, we have to do the same for `SSL_*_FILE` and co.

Reply:
done